Formula/jenkins.rb

```diff
@@ -40,8 +40,13 @@ def plist; <<~EOS
         <string>#{plist_name}</string>
         <key>ProgramArguments</key>
         <array>
-          <string>/usr/bin/java</string>
+          <string>/usr/libexec/java_home</string>
+          <string>-v</string>
+          <string>1.8</string>
+          <string>--exec</string>
+          <string>java</string>
           <string>-Dmail.smtp.starttls.enable=true</string>
+          <string>-Dhudson.model.UsageStatistics.disabled=true</string>
```

Comment:
Why this unrelated change?

Reply:
Because it's better for users to explicitly opt-in telemetry: https://wiki.jenkins.io/display/JENKINS/Usage+Statistics